README.md

```diff
@@ -87,7 +87,54 @@ Descriptors are composed together to create new descriptors with the following r
 * `configuration` are deep merged: `descriptor.configuration = _.merge({}, descriptor1.configuration, descriptor2.configuration)`
 
 
-### Stamp `options` reserved keys
+#### Priority Rules
+
+It is possible for properties to collide, between both stamps, and between different properties of the same stamp. This is often expected behavior.
+
+**Same descriptor property, different stamps:** Last in wins.
+
+**Different descriptor properties, one or more stamps:**
+
+* Optionally warn on collision at stamp creation time (set by configuration, off by default)
+* Instance properties have lowest priority
```

Comment:
It should be highest. Otherwise, there's no point passing instance to the factory.